src/main/java/in/projecteka/gateway/clients/IdentityServiceClient.java

```diff
@@ -28,6 +31,17 @@
         return getToken(loginRequestWith(clientId, clientSecret));
```

Comment:
Is the old method getting used somewhere? if not we can delete that

Reply:
yes